core/sqf/src/seatrans/tm/hbasetmlib2/src/main/java/org/trafodion/dtm/HBaseTxClient.java

```diff
@@ -694,7 +694,22 @@ public short callCreateTable(long transactionId, byte[] pv_htbldesc, Object[]  b
          LOG.error("Error while getting HTableDescriptor caused by : ", de);
          throw new IOException("Error while getting HTableDescriptor caused by : ", de);
       }
-      trxManager.createTable(ts, htdesc, beginEndKeys);
+      try {
+         trxManager.createTable(ts, htdesc, beginEndKeys);
+      }
+      catch (IOException cte) {
+         if (LOG.isTraceEnabled()) LOG.trace("HBaseTxClient:callCreateTable exception trxManager.createTable, retval: " +
+            TransReturnCode.RET_EXCEPTION.toString() +" txid: " + transactionId +" Exception: " + cte);
+         StringWriter sw = new StringWriter();
+         PrintWriter pw = new PrintWriter(sw);
+         cte.printStackTrace(pw);
```

Comment:
703-705 can be avoided. All the printStackTrace calls were removed as part of java exception handling cleanup. Instead you can LOG.error(<message>, e) methods